### ISSUE TITLE
Bot's profile page should describe it

### ISSUE THREAD

**Issue by tripleee** [Charcoal-SE/PulseMonitor#15]
The user account on SE should have a description of the bot in the profile.

**Comment by tripleee** [comment]
I tried to log in with the credentials I have on file but that brought me to [Thunder's](https://github.com/SOBotics/Thunder) [profile page](https://chat.stackoverflow.com/users/8292957/thunder). I tried to update it but I was unable to save any changes because it said the image type was wrong (presumably the profile picture).

What's the relationship between Thunder and Pulse Monitor?

Sharing a profile between two bots seems slightly weird; perhaps Halflife should simply have its own account.

**Comment by double-fault** [comment]
@tripleee when I started working on PulseMonitor, I needed an account to run the bot on. Since Thunder runs on StackOverflow only, I simply changed Thunder's profile to suit PulseMonitor on chat.SE.

Sharing account might seem slightly weird, but since PM is never gonna run on chat.SO, and Thunder never on chat.SE, there are no problems with sharing an account; technically too, sharing an account should not have any ill-effects on the bot's functionality.

If you feel like creating a separate account for PulseMonitor, go ahead! I was just too busy at the time to make an account and earn it 20 rep through edits, so I used an account I already had access to.
